### ISSUE TITLE
Privacy warning from certain URL/cache

### ISSUE THREAD

**Issue by printer83mph** [phyff/phyff#7]
big problemo

**Comment by printer83mph** [comment]
ok so i think we might have to fix this through squarespace which is where we got our domain

**Comment by l-u-n-c-h-b-o-x** [comment]
it's giving me the no after clearing my cache

**Comment by l-u-n-c-h-b-o-x** [comment]
can you email me the squarespace login?

**Comment by printer83mph** [comment]
its in my google account somewhere hold on a second

**Comment by printer83mph** [comment]
ok i have sent it

**Comment by l-u-n-c-h-b-o-x** [comment]
I think adding the www is what fucks it up in some browsers

**Comment by printer83mph** [comment]
ye seems like `https://phyff.com` works no matter what

**Comment by l-u-n-c-h-b-o-x** [comment]
https://www.phyff.com

**Comment by l-u-n-c-h-b-o-x** [comment]
yeah that's no

**Comment by printer83mph** [comment]
have not figured anything out yet, its hard to work with domains w/o breaking everything 😬

**Comment by l-u-n-c-h-b-o-x** [comment]
it still a problemo

**Comment by printer83mph** [comment]
:(

**Comment by l-u-n-c-h-b-o-x** [comment]
Any leads?

**Comment by l-u-n-c-h-b-o-x** [comment]
Squarespace still has our old site synced to our domain, I think we should disassociate the 2017 site from the squarespace domain but idk how to do that

**Comment by printer83mph** [comment]
Yeah u have to look at the older site to edit the domain settings, but its not the problem afaik

**Comment by l-u-n-c-h-b-o-x** [comment]
this is bowl shed
